controller/src/main.rs

```diff
@@ -63,6 +63,9 @@ impl SchedulingService for SchedulerService {
                     .send(Ok(status))
                     .await
                     .expect("Failed to send status to stream");
+
+                // Attendre 10 secondes avant le prochain envoi
+                sleep(Duration::from_secs(10)).await;
```

Comment:
Why? And also, please don't use french comments if the rest of the code is using english.

Reply:
done removed since it was not useful